src/components/labwareResult/LabwareResult.tsx

```diff
@@ -56,64 +58,109 @@ export default function LabwareResult({
   );
 
   const { labwareResult } = current.context;
+
   const sampleResults = mapify(labwareResult.sampleResults, "address");
+  const slotMeasurements = labwareResult.slotMeasurements
+    ? mapify(labwareResult.slotMeasurements, "address")
+    : new Map<string, SlotMeasurementRequest>();
 
   useEffect(() => {
     onChange(labwareResult);
   }, [labwareResult, onChange]);
 
+  const isMeasurementExist = "slotMeasurements" in labwareResult;
+
+  /**Ensure Tissue Coverage value is in the range 0 to 100 inclusive**/
+  const validateMeasurementField = (value: string) => {
+    const regEx = /^(^$|[0-9]|([1-9][0-9])|100)$/;
+    const val = regEx.test(value);
+    if (val) {
+      const coverage = Number(value);
+      if (coverage >= 0 && coverage <= 100) return true;
+    }
+    return false;
+  };
+
   const slotBuilder = (slot: SlotFieldsFragment): React.ReactNode => {
     return (
-      <div className="flex flex-row items-center justify-between gap-x-2">
-        <div>
-          {isSlotFilled(slot) && (
-            <div className="flex flex-row items-center justify-between">
-              <PassIcon
-                data-testid={"passIcon"}
-                className={`h-6 w-6 cursor-pointer ${
-                  sampleResults.get(slot.address)!.result === PassFail.Pass
-                    ? "text-green-700"
-                    : "text-gray-500"
-                }`}
-                onClick={() => {
-                  send({ type: "PASS", address: slot.address });
-                }}
-              />
+      isSlotFilled(slot) && (
+        <div
+          className={`flex flex-col ${
+            isMeasurementExist &&
+            labware.slots.length > 1 &&
```

Comment:
Why must labware.slots.length be greater than 1 here?

Reply:
That is for the UI style. If there are more than one slot there will be separator line displayed between each slot to make it more clear which is not needed if there is only one slot.